runtime/sema/anyresource_type.go

```diff
@@ -31,4 +31,6 @@ var AnyResourceType = &SimpleType{
 	Equatable: false,
 	// The actual returnability of a value is checked at run-time
 	ExternallyReturnable: true,
+	// The actual importability is checked at runtime
+	Importable: true,
```

Comment:
Resources may not be imported!

```suggestion
	Importable: true,
```